hyperdome_client/hyperdome_client.py

```diff
@@ -227,31 +250,31 @@ def server_switcher(self, server):
         self.message_text_field.clear()
         try:
             if self.server.is_therapist:
-                session.post(f"{self.server.url}/therapist_signup", data={"masterkey":"megumin","username":self.server.username,"password":self.server.password})
+                self.session.post(f"{self.server.url}/therapist_signup", data={"masterkey":"megumin","username":self.server.username,"password":self.server.password})
             else:
                 self.get_uid()
-                self.therapist = session.post(f"{self.server.url}/request_therapist",  data={"guest_id":self.uid}).text
+                self.therapist = self.session.post(f"{self.server.url}/request_therapist",  data={"guest_id":self.uid}).text
                 if self.therapist:
                     self.is_connected = True
-        except:
+        except Exception as e:
+            print (e)
```

Comment:
```suggestion
            print(''.join(traceback.format_exception(type(e), e, e.__traceback__)))
```
This is how you get the full trace